config/themes/techbook/theme.css

```
@@ -364,22 +364,30 @@ p:has(+ table) {
 
 /* 表番号：所属セクションに応じた形式 */
 /* 章直下（section.level1 > table）*/
-section.level1 > p:has(+ table)::before {
+section.level1 > p:has(+ table)::before,
+section.level1 .column > p:has(+ table)::before,
+section.level1 .tips > p:has(+ table)::before {
   content: "表" counter(chapter) ".-" counter(table) ": ";
 }
 
 /* h2直下（section.level2 > table）*/
```

Stylelint (17.5.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)